### ISSUE TITLE
Error when trying to boot up docker image

### ISSUE THREAD

**Issue by mockdeep** [EBWiki/EBWiki#3872]
**Describe the bug**

I'm unable to use the docker build to boot the app on my local machine (Linux). Using the instructions from [this guide](https://github.com/EBWiki/EBWiki/blob/main/docs/SETUP_LOCALLY.md), I see the following error:

```sh
$ docker pull ebwiki/ebwiki
Using default tag: latest
latest: Pulling from ebwiki/ebwiki
Digest: sha256:2be98bcc4eafc2d32330c8367d175632c2699b516f9f6c2c4b2a539b032050dc
Status: Image is up to date for ebwiki/ebwiki:latest
docker.io/ebwiki/ebwiki:latest
$ make run
docker run --detach --rm --volume "/home/fletch/Sync/projects/open_source/EBWiki":/usr/src/ebwiki \
	--publish 3000:3000 --name ebwiki ebwiki/ebwiki
2121c3f3937a47548954785ba296e42f82d31addef9d44b2c4adbdc593ab8d3a
./dev_provisions/prewarm.sh
## Warming up the server...
.......Error: No such container: ebwiki

## Ending warm up because the server took too long or stopped unexpectedly.

## To debug, start the server with the following commands and open an issue with the output:
./dev_provisions/prewarm.sh: line 17: PWD: command not found
docker run --volume "":/usr/src/ebwiki --publish 3000:3000 --name ebwiki ebwiki/ebwiki

## You can also use these commands to build the server image from scratch and then run it:
docker container rm ebwiki
docker image rm ebwiki/ebwiki:latest
docker build --tag ebwiki/ebwiki .
./dev_provisions/prewarm.sh: line 23: PWD: command not found
docker run --volume "":/usr/src/ebwiki --publish 3000:3000 --name ebwiki ebwiki/ebwiki
Makefile:5: recipe for target 'run' failed
make: *** [run] Error 7
```

**To Reproduce**
1. Clone EBWiki
2. run `docker pull ebwiki/ebwiki`
3. run `make run`

**Expected behavior**
Development server boots.

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by mockdeep** [comment]
Looks like the same issue is [happening on CI](https://github.com/EBWiki/EBWiki/runs/1718400552?check_suite_focus=true). I'm not too familiar with Docker, but I'd be willing to work on getting the build to pass on CI without it if the maintainers are okay with that.

**Comment by rlgreen91** [comment]
@mockdeep Thanks for listing this here.  I let @managedkaos know as he is the one who initially created the action.  I'm also wondering if this actually stems from the fact that you are running on a fork.  I'll add you as a contributor here, and that way you can create a branch and try pushing directly from there rather than your fork.

**Comment by mockdeep** [comment]
@rlgreen91 thanks for adding me as contributor! I made a new PR and unfortunately [see the same issue](https://github.com/EBWiki/EBWiki/runs/1719117796?check_suite_focus=true). I'll poke around with it and see if I can fix it.

**Comment by managedkaos** [comment]
This issue is fixed by #3884
